bundled/tool/check_consent.py

```diff
@@ -0,0 +1,32 @@
+from common import update_sys_path
+from pathlib import Path
+import os
+import sys
+import json
+
```

Comment:
I think you accidentally removed 

```
from kedro_telemetry.plugin import _check_for_telemetry_consent
from kedro_telemetry.plugin import (
    _get_project_properties,
    _get_or_create_uuid,
) 
```

Reply:
https://github.com/kedro-org/vscode-kedro/pull/84/#discussion_r1750041770

I think this is the reason, I have added it back.

Reply:
@noklam sorry I added the half code as code suggestion. but if I am not wrong, we need to keep `update_sys_path` before we import `kedro_telemetry`

```
from common import update_sys_path
from pathlib import Path
import os
import sys
import json

update_sys_path(
    os.fspath(Path(__file__).parent.parent / "libs"),
    os.getenv("LS_IMPORT_STRATEGY", "useBundled"),
)

from kedro_telemetry.plugin import _check_for_telemetry_consent
from kedro_telemetry.plugin import (
    _get_project_properties,
    _get_or_create_uuid,
)

# Rest of the code
```